docs/tutorial.rst

```diff
@@ -875,62 +876,54 @@ Again, methods to do this are discussed in the
 Incorporating selection
 =======================
 
-Here are some examples of how to incorporate selection.
-This API is **not final**, and will probably change!
+There two general ways to incorporate selection into a simulation.
+Currently, both only work using the SLiM engine.
+We can specify a :class:`distribution of fitness effects <.DFE>`
+for all new mutations
+(as in `the first example <sec_tute_genome_wide_dfe>`_)
+and optionally a subset of the genome on which these occur
+(as in `the second example <sec_tute_selection_single_gene>`_),
+which can even be obtained from an :class:`.Annotation`
+(see `the third example <sec_tute_selection_annotation>`_),
+Additionally, we add a single mutation under selection,
+as for instance in a `selective sweep <sec_tute_selective_sweep>`_.
+
+
+.. _sec_tute_genome_wide_dfe:
 
 1. Simulating with a genome-wide DFE
 ------------------------------------
 
-In this example, we'll simulate
-Gutenkunst et al. HomSap/OutOfAfrica_3G09 model, with a DFE.
 To add mutations under selection to a simulation,
 we need to tell the contig where to put the mutations,
 and what distribution of selection coefficients they will have.
-This is set up by :meth:`.contig.add_genomic_element_type`,
-which needs to know which ``intervals`` the mutations will apply to,
-a list of :class:`.ext.MutationType` objects,
-and the ``proportions`` of all mutations that will come from each of these mutation types.
-The specification mirrors that of SLiM,
-and MutationTypes have the same arguments as in SLiM.
+To do this, we
 
-.. code-block:: python
+- choose a distribution of fitness effects (a :class:`.DFE`),
+- choose an :class:`.Annotation`, and
+- add these to the :meth:`Contig <.Contig.add_DFE>`,
```

Comment:
wait, but in this example we are not choosing an annotation, we are saying to add the DFE to the whole contig.

Reply:
ah good point